### ISSUE TITLE
Track numbers lower than 10 cannot be parsed

### ISSUE THREAD

**Issue by agersant** [polyfloyd/rust-id3#73]
Steps to reproduce:

1. Read the id3 tag from the attached example file `9.mp3`
2. Call `tag.track()` on the resulting tag
3. Observe that the track number comes back as `None`, even though the `TRCK` frame has the value `9`.

Sample code:
```
let tag = Tag::read_from_path("testdata/10.mp3").unwrap();
assert_eq!(tag.track(), Some(10)); // ✔
let tag = Tag::read_from_path("testdata/9.mp3").unwrap();
assert_eq!(tag.track(), Some(9)); // ❌
```

Looking into this a tiny bit deeper, I noticed that the frame text is read as `9\u{0}` instead of just `9`, which makes the parsing fail.

I am also attaching a file with track number `10`, which works fine (for comparison). Both files were tagged using `foobar2000 v1.6.6` on Windows. Also note that specifying a value for `total tracks` makes this bug go away.

[track numbers.zip](https://github.com/polyfloyd/rust-id3/files/7512077/track.numbers.zip)

**Comment by polyfloyd** [comment]
Hi agersant,

Thank you for the detailed report :) Yes, this seems erroneous behaviour. Will check it out

**Comment by polyfloyd** [comment]
Alright, I pushed a fix.

ID3 specifies that track fields can use / to separate track and the total number of tracks. But the spec also states that a null byte may be used, although this is more intended for other text fields according to my interpretation.

I have updated the `track()` function to support both, that should take care of this.

**Comment by polyfloyd** [comment]
Released v0.6.6

**Comment by agersant** [comment]
Wow that was a quick turnaround time, thanks a lot! ❤
